### ISSUE TITLE
Team donation gives error RECEIVING_PREFERENCE_MANDATES_MANUAL_ACTION 

### ISSUE THREAD

**Issue by jelv** [liberapay/liberapay.com#1324]
On my ledger it says:

November 5, 2018
anonymous donation for your role in the Liberapay team (25 months of £0.20) — pending — error message: RECEIVING_PREFERENCE_MANDATES_MANUAL_ACTION

A bit unclear for me the user. Not clear if I should take action or this will be fixes with some manual action by admin?

**Comment by eldelacajita** [comment]
+1 to this. I'm having the same issue while trying to donate to an individual account. A confirmation email by Paypal says the transaction was successful. What happened to the money?

**Comment by trebmuh** [comment]
(a shot in the dark here) Wouldn't that be that one needs to connect to his paypal account to accept the donation since it's not a currency set up has _automatically accepted_ in his paypal account?

**Comment by eldelacajita** [comment]
Update: now I'm not seeing that error anymore, but the donation wasn't made effective, still appears as pending renovation (while the money was effectively sent via Paypal).

**Comment by jelv** [comment]
I got the error again. I had this the other way when I got a donation from Liberapay team. The receiver has to accept the payment in PayPal. Maybe it was because of currency conversion? I don't know for sure. After that it completes. 

For the giver it looks like this.
![screenshot_4](https://user-images.githubusercontent.com/7106279/51168618-1063e580-18aa-11e9-8768-9af1916377e1.png)
![screenshot_3](https://user-images.githubusercontent.com/7106279/51168619-1063e580-18aa-11e9-920b-a2c8e50ffdfa.png)
